src/TitleDetails/DeletePieceModal/DeletePieceModalContainer.js

```diff
@@ -0,0 +1,78 @@
+import { useEffect } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Button } from '@folio/stripes/components';
+import { ModalFooter } from '@folio/stripes-acq-components';
+
+import { DeletePieceModal } from './DeletePieceModal';
+import { useHoldingItems } from '../hooks';
+
+const DeletePieceModalContainer = ({
+  onCancel,
+  onConfirm,
+  piece,
+  setIsLoading,
+}) => {
+  const { itemId, holdingsRecordId } = piece;
+  const { itemsCount, isFetching } = useHoldingItems(holdingsRecordId, { searchParams: { limit: 1 } });
+  const isLastConnectedItem = itemId && itemsCount === 1;
```

Comment:
what if we saved piece, created holding without item and decided to remove?

Reply:
As I understand it, if a piece is not related to an item, we show default confirm modal and delete only the piece.

Reply:
why not to remove holding? the idea is not to keep empty holdings

plz ask Dennis in the ticket